protokube/pkg/protokube/hetzner_volume.go

```diff
@@ -0,0 +1,119 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package protokube
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"os"
+	"strconv"
+
+	"github.com/hetznercloud/hcloud-go/hcloud"
+	"github.com/hetznercloud/hcloud-go/hcloud/metadata"
+	"k8s.io/klog/v2"
+	"k8s.io/kops/protokube/pkg/gossip"
+	gossiphetzner "k8s.io/kops/protokube/pkg/gossip/hetzner"
+	"k8s.io/kops/upup/pkg/fi/cloudup/hetzner"
+)
+
+// HetznerVolumes defines the Hetzner Cloud volume implementation.
+type HetznerVolumes struct {
+	hcloudClient *hcloud.Client
+	server       *hcloud.Server
+}
+
+func (h HetznerVolumes) AttachVolume(volume *Volume) error {
+	//TODO implement me
+	panic("implement me")
```

Comment:
I take it we don't need this because protokube doesn't attach volumes any more, because we're using etcd-manager.  If so, we could tweak the comment to say that.